### DISCUSSION TITLE
Beginner Friendly Project to contribute

### DISCUSSION THREAD

**Discussion by maha885** [cncf/mentoring#1090] [General]
What is the most beginner friendly project to contribute in CNCF landscape

**Comment by nate-double-u** [comment]
Hi @maha885, we have a tool we've been developing (currently in alpha) to help folks find projects looking for contributors: [CLOTributor](https://clotributor.dev/). It's useful for helping contributors find projects that suit their skills and interests best.

I also like to suggest @Riaankl's article: "[Do you want to start contributing to open source and need help figuring out where to begin?](https://contribute.cncf.io/contributors/getting-started/)" It's full of great suggestions about how to get started in open source.
